frontend/src/app/Settings/Facility/FacilityFormContainer.tsx

```diff
@@ -225,6 +229,7 @@ const FacilityFormContainer: any = (props: Props) => {
     if (appInsights) {
       appInsights.trackEvent({ name: "Save Settings" });
     }
+    setUpdatedFacility(() => facility);
     const provider = facility.orderingProvider;
     const saveFacility = props.facilityId ? updateFacility : addFacility;
     await saveFacility({
```

Comment:
We will need to save created facilities in the Redux store _with_ their primary key IDs generated by the database. What I think this will look like is:

1. Updating the `addFacility`/`updateFacility` mutations to return `Facility` and its necessary subfields (including `internalId`)
2. Save the resolved value of the `saveFacility` call in `FacilityFormContainer.tsx` to a variable
3. Write that variable to local state as `updatedFacility`, and dispatch _that_ piece of state to the store

Reply:
**Note:** resolved in 0102220